### ISSUE TITLE
lib system not found in /lib64

### ISSUE THREAD

**Issue by thiteixeira** [zhehaowang/ndnfs-port#8]
Hi,

Configuration fails when trying to `./waf configure`
My environment is a Ubuntu 14.04 with NFD, ndn-cxx, ndn-cpp, ndn-tools, and Boost libraries installed. In all of the above, the step `Checking boost libs` works. On the ndnfs-port I get the error `lib system not found in /lib64`.

Do you have a fix for this issue please?

Thanks in advance

**Comment by cawka** [comment]
You may have multiple versions of boost libraries installed.  I would recommend removing all manually installed ones.

In some cases, you also need to explicitly specify where the libraries are installed (this still can create issues):

```
 ./waf configure --boost-libs=/path/to/folder/with/lib
```

**Comment by thiteixeira** [comment]
Hi Alex,
Thanks for the quick reply.
I checked my Boost libraries with
`$ dpkg -S /usr/include/boost/version.hpp`
`libboost1.54-dev: /usr/include/boost/version.hpp`

Uninstalled with
`$ sudo apt-get autoremove libboost1.54-dev`

And reinstalled again with
`$ sudo apt-get install libboost-all-dev`

After getting the same error, I tried these instructions:
In /usr do:
`sudo ln -s lib/x86_64-linux-gnu/ lib64`
and then it works:
`./waf configure --boost-libs=/usr/lib64`
and it worked.

Thanks again.
